src/RGB/RGBColor.php

```diff
@@ -16,27 +16,27 @@ public function __construct(
 
     public function toHSV(): HSVColor
     {
-        $percentageRed   = ($this->red / 255);
-        $percentageGreen = ($this->green / 255);
-        $percentageBlue  = ($this->blue / 255);
+        $percentageRed   = $this->red / 255;
+        $percentageGreen = $this->green / 255;
+        $percentageBlue  = $this->blue / 255;
 
         $highestRGBPercentage = max($percentageRed, $percentageGreen, $percentageBlue);
         $lowestRGBPercentage  = min($percentageRed, $percentageGreen, $percentageBlue);
         $chroma               = $highestRGBPercentage - $lowestRGBPercentage;
 
-        $value = 100 * $highestRGBPercentage;
-        if ((int) round($chroma) === 0) {
-            return new HSVColor(0, 0, (int) round($value));
+        $highestRGB = (int) round(100 * $highestRGBPercentage);
+        if ($chroma < 0.5) {
```

Comment:
🧠 This is brain-intensive but simple!

Reply:
Hi, thanks for your contribution! This line of code contained a bug resulting in all HSV colors not having a hue or saturation values. Can you rebase on master? The rest of the changes seem like a good idea!

Reply:
Done 🍏